tests/rptest/clients/rpk.py

```diff
@@ -360,7 +360,15 @@ def parse_partition(string):
 
         def try_describe_group(group):
             cmd = ["describe", group]
-            out = self._run_group(cmd)
+            try:
+                out = self._run_group(cmd)
+            except RpkException as e:
+                if "COORDINATOR_NOT_AVAILABLE" in e.msg:
+                    # Transient, return None to retry
+                    return None
+                else:
+                    raise
+
```

Comment:
Should this retry be better implemented at a lower level?

Currently `rpk` (via `franz-go`) retries the `MetadataRequest` call once if its reply is missing the coordinator. Would it be better if it retries more than once, with probably an `rpk` paratemeter providing a timeout value for retrying?

Reply:
Yes, I would prefer rpk to be a bit more friendly do own some level of retries when it comes to this case: currently rpk generally passes through transient leaderless-ness or results of requests to a node that was no longer the leader -- this comes up in e.g. 'rpk topic describe' output as well as consumer groups, and it's increasingly common now that we have the leader balancer, data balancer etc.

Not necessarily actionable right now, but perhaps something for @r-vasquez to contemplate.  Current behaviour is not wrong exactly, but we might want to broaden the range of situations that we classify as retryable.